### DISCUSSION TITLE
How to set up credentials

### DISCUSSION THREAD

**Discussion by XiaoguangGuo** [saleweaver/python-amazon-sp-api#220] [Feature requests]
I'm a new. 
How can I set up credentials?  who can give me a step-to-step guide? I have all the passwords. Can I write the followings and save them as a yml file? 
"
default:
  refresh_token: ''
  lwa_app_id: ''
  lwa_client_secret: ''
  aws_secret_key: ''
  aws_access_key: ''
  role_arn: ''
"

**Comment by braeutigamj** [comment]
There are multiple options: https://python-amazon-sp-api.readthedocs.io/en/latest/credentials/
Yes you can create a yaml file: https://python-amazon-sp-api.readthedocs.io/en/latest/config_file/
